### DISCUSSION TITLE
Improved background sync on Android

### DISCUSSION THREAD

**Discussion by rcsilva83** [ente-io/ente#973] [Enhancements]
I have a Xiaomi Poco F1 (MIUI 12.0.3 - Android 10) and sync is only working with the app opened. If I switch to other app, after some time I reopen Ente app and the sync was interrupted.

I've already disabled MIUI's battery saving option but it is still not working....

**Comment by mnvr** [comment]
Hey!

If it is fine with you, could you please send logs to our support email from your app by clicking on "Report Bug" from the Settings screen? This might be an Xiaomi + Android 10 specific issue, so by taking a look at the logs we might be able to figure out if there is something we can do about it.

I would also encourage you to see if you can try out the steps listed here 

- https://dontkillmyapp.com/xiaomi

These manufacturers do all sorts of undocumented things which sometimes make it very hard to run in the background reliably unfortunately 😞, so it'll be great if you could double check that you've changed all the known settings to let your phone know that it is okay to let ente run.

**Comment by rcsilva83** [comment]
> If it is fine with you, could you please send logs to our support email from your app by clicking on "Report Bug" from the Settings screen?

Hi, @mnvr ! I did this yesterday and sent the e-mail with the logs attached.

**Comment by xthrk** [comment]
Are there any updates here? Unfortunately, the background sync is still not working. The app must be actively open in the foreground for synchronization to work.

**Comment by ua741** [comment]
Hey @xthrk,

We are planning to focus on improving background sync and building support for resumable uploads early next year. Before that, we hope to ship Legacy, enhance search, and improve the overall performance of the app.

In the meantime, could you please report any bugs from the app? We will take a look at the logs to see if there's anything unusual.

**Comment by ghost** [comment]
Any updates on this?

**Reply by brog-io** [reply]
Resumable updates should be available next week, this will improve uploading
